guides/source/action_mailer_basics.md

```diff
@@ -422,6 +422,27 @@ use the rendered text for the text part. The render command is the same one used
 inside of Action Controller, so you can use all the same options, such as
 `:text`, `:inline` etc.
 
+In the case that you have a Rails application with an unorthodox file organization and wish to render a template located outside of the default `app/views/mailer_name/` directory, you can apply the `prepend_view_path`, like so:
+
+```ruby
+class UserMailer < ApplicationMailer
+  default from: 'notifications@example.com'
+
+  def welcome_email
+    @user = params[:user]
+    full_template_path = Rails.root.to_s + "/custom_folder/user_mailer/views/"
+    prepend_view_path(full_template_path)
+    mail(
+      to: @user.email,
+      subject: subject,
+      template_path: full_template_path,
```

Comment:
You don't need the full path here anymore, it can be relative.

Reply:
You mean after you call the `prepend_view_path` method in the class, you don't have to specify the template_path?

Reply:
yes. The template path is now both `app/views` and `full_template_path`, so Rails will search on both.